board/views.py

```diff
@@ -1,145 +1,32 @@
-        }
-        tweet_list = self.api_request('tweets/search/recent', query)
-        return [item['id'] for item in tweet_list]
-
-    def get_user_by_id(self, user_id):
-        query = {
-            'ids': user_id,
-            'user.fields': 'id,username,url'
-        }
-        return self.api_request('users', query)[0]
-
-    def _compose_request(self, query):
-        if not query:
-            raise Exception('No query provided!')
-        request_params = {
-            'headers': {
-                'Authorization': f'Bearer {self.bearer_token}',
-            }
-        }
-
-        if isinstance(query, dict):
-            request_params['params'] = query
-            return request_params
-
-        raise TypeError('Incorrect type of argument "query"')
-
-
-class IGDB:
-    def __init__(self, client_id, auth_token):
-        self.client_id = client_id
-        self.auth_token = auth_token
-
-    def api_request(self, endpoint, query):
-        url = IGDB._build_url(endpoint)
-        params = self._compose_request(query)
-        response = post(url, **params)
-        response.raise_for_status()
-
-        return response.json()
-
-    def get_games(self, query):
-        return self.api_request('games', query)
-
-    def get_genres(self, query):
-        return self.api_request('genres', query)
-
-    def get_platforms(self, query):
-        return self.api_request('platforms', query)
-
-    @staticmethod
-    def _build_url(endpoint=''):
-        return f'{API_IGDB_URL}{endpoint}/'
-
-    def _compose_request(self, query):
-        if not query:
-            raise Exception('No query provided!')
-        request_params = {
-            'headers': {
-                'Client-ID': f'{self.client_id}',
-                'Authorization': f'Bearer {self.auth_token}',
-            }
-        }
-
-        if isinstance(query, dict):
-            request_params['params'] = query
-            return request_params
-
-        raise TypeError('Incorrect type of argument "query"')
-
-
-twitter_wrapper = Twitter(os.getenv('BEARER_TOKEN'))
-igdb_wrapper = IGDB(os.getenv('API_CLIENT_ID'), os.getenv('API_SECRET_KEY'))
-
-
-def get_img_url(image_id, size='screenshot_big'):
-    return 'https://images.igdb.com/igdb/image/upload/t_{}/{}.jpg'.format(size, image_id)
+twitter_wrapper = TwitterWrapper(BEARER_TOKEN)
+igdb_wrapper = IgdbWrapper(API_CLIENT_ID, API_SECRET_KEY)
 
 
 class Game:
     def __init__(self, game_id):
-        params = {
-            'fields': 'id, cover.image_id, name, '
-                      'summary, screenshots.image_id, '
-                      'genres.name, release_dates, '
-                      'platforms.name, '
-                      'aggregated_rating, aggregated_rating_count, '
-                      'rating, rating_count, slug ',
-            'filter[id][eq]': game_id
-        }
-
-        res = igdb_wrapper.get_games(params)[0]
+        res = igdb_wrapper.get_games_by_id(game_id)[0]
```

Comment:
What if there is no game found by the id?

Reply:
I send only valid game_id to this class. And before that I have got a list of game_ids in "main" function:
`res = igdb_wrapper.get_games_id(params)`
Shood I check it out in `class Game`?